produce/lib/produce/options.rb

```diff
@@ -65,6 +65,13 @@ def self.available_options
                                      description: "Skip the creation of the app on iTunes Connect",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :itc_users,
+                                     short_option: "-s",
+                                     env_name: "ITC_USERS",
+                                     optional: true,
+                                     type: Array,
+                                     description: "Array with Itunes Connect Users for Limit User Access",
```

Comment:
This text isn't clear directly, can we rephrase to something like `Array of iTunes Connect users that have access to newly created app? If provided, no other users will have access` (at least that's how I understood it, is that correct?)